### PR TITLE
Token choices UI consistency

### PR DESCRIPTION
**Pull request by jackburrus**
This PR removes the TokenChoices component in favor of the TokenPicker component for UI consistency.
image

### CODE REVIEW

File: apps/hyperdrive-trading/src/ui/hyperdrive/longs/CloseLongForm/CloseLongForm.tsx [hunk 1]
```diff
@@ -41,6 +43,18 @@ export function CloseLongForm({
...
+    account,
+    tokenAddress: baseToken.address,
+    decimals: baseToken.decimals,
+  });
+
+  const { balance: sharesTokenBalance } = useTokenBalance({
+    account,
+    tokenAddress: sharesToken.address,
+    decimals: sharesToken.decimals,
+  });
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/ui/hyperdrive/longs/CloseLongForm/CloseLongForm.tsx:56]: Not used

File: apps/hyperdrive-trading/src/ui/hyperdrive/lp/RemoveLiquidityForm/RemoveLiquidityForm.tsx [hunk 2]
```diff
@@ -45,6 +47,18 @@ export function RemoveLiquidityForm({
...
+    account,
+    tokenAddress: baseToken.address,
+    decimals: baseToken.decimals,
+  });
+
+  const { balance: sharesTokenBalance } = useTokenBalance({
+    account,
+    tokenAddress: sharesToken.address,
+    decimals: sharesToken.decimals,
+  });
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/ui/hyperdrive/lp/RemoveLiquidityForm/RemoveLiquidityForm.tsx:61]: Not used

File: apps/hyperdrive-trading/src/ui/hyperdrive/shorts/CloseShortForm/CloseShortForm.tsx [hunk 3]
```diff
@@ -40,6 +43,17 @@ export function CloseShortForm({
...
+    account,
+    tokenAddress: baseToken.address,
+    decimals: baseToken.decimals,
+  });
+
+  const { balance: sharesTokenBalance } = useTokenBalance({
+    account,
+    tokenAddress: sharesToken.address,
+    decimals: sharesToken.decimals,
+  });
```
Comment (DannyDelott) [inline apps/hyperdrive-trading/src/ui/hyperdrive/shorts/CloseShortForm/CloseShortForm.tsx:56]: Not used

**Review by DannyDelott — APPROVED** [review]
Nice!

File: apps/hyperdrive-trading/src/ui/hyperdrive/longs/CloseLongForm/CloseLongForm.tsx (hunk 1 above)
Comment (jackburrus) [reply apps/hyperdrive-trading/src/ui/hyperdrive/longs/CloseLongForm/CloseLongForm.tsx:56]: Ah, my bad. Thank you

**Pull request merged**